django/db/models/options.py

```diff
@@ -744,16 +737,38 @@ def get_fields(self, include_parents=True, include_hidden=False):
         - include_hidden:  include fields that have a related_name that
                            starts with a "+"
         """
+        if include_parents is False:
+            include_parents = PROXY_PARENTS
         return self._get_fields(include_parents=include_parents, include_hidden=include_hidden)
 
     def _get_fields(self, forward=True, reverse=True, include_parents=True, include_hidden=False,
-                    export_ordered_set=False):
+                    seen_models=None):
+        """
+        Internal helper fuction to return fields of the model. If forward=True then fields defined
+        on this model are returned. If reverse is True, then relations pointing to this model
+        are returned. The include_hidden fields control returning fields with is_hidden attribute
```

Comment:
I think you mean "The include_hidden argument toggles if fields with is_hidden=True are included."?